html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<style>
+  body {
+    background-image: linear-gradient(to right, rgb(182, 31, 182) , rgba(0, 0, 255, 0.836));
+  }
+  div {
+    font-weight: 600;
+  }
+</style>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <div>
+      <div>Email</div>
+      <input type="email" id="email">
+    </div>
+
+    <div>
+      <div>Why is coding cool?</div>
```

Comment:
This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!